PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/PBMCore/NativeAssets/NativeAdResponse/NativeAdAssetBoxingError.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  NativeAdAssetBoxingError.swift
+//  PrebidMobileRendering
+//
+//  Copyright © 2021 Prebid. All rights reserved.
+//
+
+import Foundation
+
+enum NativeAdAssetBoxingError {
+    static var noDataInsideNativeAdMarkupAsset: Error
+    {
```

Comment:
Let's move this bracket on the line above.
here and in all next cases in this file